src/GitHub.InlineReviews/Views/CommentView.xaml

```diff
@@ -50,31 +50,46 @@
                 </Style>
             </StackPanel.Style>
 
-            <StackPanel Orientation="Horizontal">
+            <DockPanel>
+
+              <StackPanel Orientation="Horizontal" DockPanel.Dock="Left" >
                 <controls:AccountAvatar Width="16" 
-                                        Height="16" 
-                                        Account="{Binding User}"/>
+                                Height="16" 
+                                Account="{Binding User}"/>
 
                 <TextBlock Foreground="{DynamicResource GitHubVsToolWindowText}" FontWeight="Bold" Text="{Binding User.Login}" Margin="4 0"/>
                 <ui:GitHubActionLink Content="{Binding UpdatedAt, Converter={ui:DurationToStringConverter}}"
-                                     Command="{Binding OpenOnGitHub}"
-                                     Foreground="{DynamicResource GitHubVsToolWindowText}"
-                                     Opacity="0.75" />
+                             Command="{Binding OpenOnGitHub}"
+                             Foreground="{DynamicResource GitHubVsToolWindowText}"
+                             Opacity="0.75" />
                 <Border Background="{DynamicResource VsBrush.InfoBackground}"
                         BorderBrush="{DynamicResource VsBrush.AccentPale}"
                         BorderThickness="1"
                         CornerRadius="3"
                         Padding="2 1"
                         Visibility="{Binding IsPending, Converter={ui:BooleanToVisibilityConverter}, FallbackValue=Collapsed}">
-                    <TextBlock FontSize="10">Pending</TextBlock>
+                  <TextBlock FontSize="10">Pending</TextBlock>
                 </Border>
-            </StackPanel>
+              </StackPanel>
+
+              <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" DockPanel.Dock="Right"
+                          Visibility="{Binding CanEditOrDelete, Converter={ui:BooleanToVisibilityConverter}}">
+                <ui:GitHubActionLink Content="Edit"
```

Comment:
![image](https://user-images.githubusercontent.com/11719160/40435361-665baaf2-5ea8-11e8-9208-e83d9be0e04a.png)

![image](https://user-images.githubusercontent.com/11719160/40435296-46bf473a-5ea8-11e8-84b6-2f45d9f8918a.png)

Rather than `Edit`, should this be `Update comment` like on .com? 👆

Reply:
I think this references the Edit / Delete links in the screenshot above?

![Screenshot of feature](https://user-images.githubusercontent.com/417571/40386540-fc25b5a4-5dd7-11e8-9f73-4867e6f22063.png)

If so, I think it's okay to leave this the way how it is. However, I'm 👍 for having the submit button text be "Update comment" like the dotcom screenshot you've posted.

Reply:
Fixed up.